CMakeLists.txt

```diff
@@ -80,3 +76,16 @@ elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
 	set_source_files_properties(${SG14_TEST_SOURCE_DIRECTORY}/plf_colony_test.cpp PROPERTIES
 		COMPILE_FLAGS "/wd4127") # Disable conditional expression is constant, use if constexpr
 endif()
+
+install(TARGETS ${PROJECT_NAME} EXPORT ${PROJECT_NAME}_targets)
+
+include(GNUInstallDirs)
+
+install(EXPORT ${PROJECT_NAME}_targets
+    NAMESPACE ${PROJECT_NAME}::
+    FILE ${PROJECT_NAME}-config.cmake
+    DESTINATION "${CMAKE_INSTALL_DATADIR}/cmake/${PROJECT_NAME}"
+)
+
+install(DIRECTORY "${PROJECT_SOURCE_DIR}/SG14" DESTINATION include)
```

Comment:
May want to use `${CMAKE_INSTALL_INCLUDEDIR}` install of baking `include`.
Should use `${SG14_INCLUDE_DIRECTORY }` instead of hardcoding `${PROJECT_SOURCE_DIR}/SG14}`.

Reply:
Noted.